qiskit_dynamics/pulse/pulse_to_signals.py

```diff
@@ -79,14 +81,60 @@ def __init__(
         self._carriers = carriers or {}
 
     def get_signals(self, schedule: Schedule) -> List[DiscreteSignal]:
-        """
+        r"""Convert a schedule to a corresponding list of DiscreteSignal instances.
```

Comment:
:class:

Reply:
I actually purposefully left this without `:class:` based on a rule I try to follow that the first line should be readable as plain text for when users call `help`. I'm sure this isn't followed everywhere in Dynamics.